test/test.cpp

```diff
@@ -9,12 +9,5 @@
 // https://github.com/catchorg/Catch2/blob/v2.x/docs/ci-and-misc.md#precompiled-headers-pchs
 #undef TWOBLUECUBES_SINGLE_INCLUDE_CATCH_HPP_INCLUDED
 #define CATCH_CONFIG_IMPL_ONLY
-#define CATCH_CONFIG_RUNNER
+#define CATCH_CONFIG_MAIN
```

Comment:
I'd like to keep our custom main so we can eventually re-route logging to the Catch2 test logging mechanisms